config/routes.rb

```diff
@@ -1,14 +1,20 @@
 Rails.application.routes.draw do
-  devise_for :users
+  
+    devise_for :users, controllers: {
+      sessions: 'users/sessions',
+      registrations: 'users/registrations'
+  }
+  get '/member-data', to: 'members#show'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   root "users#index"
   resources :users, only: [:index, :show] do
```

Comment:
### Optional
- [ ] It is often a good practice to add `/api` to URL or `/api/v1` with version number